assets/naf-process-chile-aquaculture.sql.j2

```diff
@@ -0,0 +1,12 @@
+SELECT
+  PARSE_TIMESTAMP("%Y%m%d%k%M%S", CONCAT(DA,TI)) timestamp,
+#  RC AS callsign,
+  NA AS shipname,
+  IR AS internal_id,
```

Comment:
@smpiano for CHile lets also add the `XR` as `external_id`. It's storing weird data, but we may want that data.